### ISSUE TITLE
Upload issues if Cloudfare is used as Reverse Proxy (100MB limit set by Cloudfare) - Split upload into 100MB Chunks

### ISSUE THREAD

**Issue by FurkanVG** [ShaneIsrael/fireshare#245]
**Describe the bug**
If Cloudfare is being used for Fireshare as DNS Reverse Proxy, the limit of upload is limit to 100MB (Default Free Plan). This means that no upload of files is possible more than 100MB.

**To Reproduce**
1. Use domain behind Cloudflare proxy free plan (this enforces 100 MB upload limit)'
2. Try to upload file more than 100MB to fireshare
3. See error

**Expected behavior**
Upload works without errors

**Screenshots**
![image](https://github.com/ShaneIsrael/fireshare/assets/68749298/8ccb038e-6419-446a-8d92-d7ebd0513a57)

**Desktop (please complete the following information):**
 - Browser Firefox
 
**Additional context**
Im no Expert in NGINX or Webservers, but i read in forums that is possible to "split" the upload in chunks, means if a filesize is bigger than 100MB, lets say 800 MB, fireshare could split these into 100MB chunks so an upload is still possible even if there is an limit of 100MB.
Correct me if im wrong or anything, just realized that this is my current issue. If i disable the Proxy from Cloudfare, then it works fine because then my NGINX conf is active and i can set on my own the upload limit.
Thanks!

**Comment by FurkanVG** [comment]
1min after creation of this ticket, i saw https://github.com/ShaneIsrael/fireshare/issues/187 ...
Any update on this topic? :D

**Comment by ShaneIsrael** [comment]
This would be a feature enhancement, not a bug since upload works fine. This is something I do want to eventually address but it may be a while as I tend to focus primarily on actual bugs or features that would directly benefit my own personal use case. Just depends on whether I have a free day and in the mood to code some general improvements. 

That being said, it is open source and available for anybody to tackle if they do not want to wait on me.
